### PR TITLE
MAUI-1550 Add the ItemsSource support in the SfTabView UG 

### PR DESCRIPTION
**Pull request by jeya-kasipandi**
Added itemsSource support in the SfTabView UG.

### CODE REVIEW

File: MAUI/Badge-View/Badge-customization.md [hunk 1]
```diff
@@ -298,3 +298,49 @@ Content = sfBadgeView;
 
 ![.NET MAUI Badge View Corner Radius](badge-customization_images/net_maui_badge_notifications_corner_radius.png)
 
+## Alignment of badge
+
+You can align the badge view using the Center, Start, and End properties of BadgeAlignment property.
```
Comment (Choza-rajan) [inline MAUI/Badge-View/Badge-customization.md:303]: Align the badge view using the Center, Start, and End properties of BadgeAlignment property.

File: MAUI/Badge-View/Position-customization.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ documentation: ug
...
+The default position of notification is [`TopRight`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgePosition.html#Syncfusion_Maui_Core_BadgePosition_TopRight). The position can be customized as [`TopLeft`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgePosition.html#Syncfusion_Maui_Core_BadgePosition_TopLeft),  [`BottomLeft`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgePosition.html#Syncfusion_Maui_Core_BadgePosition_BottomLeft),  [`BottomRight`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgePosition.html#Syncfusion_Maui_Core_BadgePosition_BottomRight), `Left`,`Top`, `Right`, and `Bottom` using the [`Position`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgeSettings.html#Syncfusion_Maui_Core_BadgeSettings_Position) property.
```
Comment (laksmipriyamurugesan) [inline MAUI/Badge-View/Position-customization.md:12]: The default position of notification is [`TopRight`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgePosition.html#Syncfusion_Maui_Core_BadgePosition_TopRight). The position can be customized as [`TopLeft`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgePosition.html#Syncfusion_Maui_Core_BadgePosition_TopLeft),  [`BottomLeft`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgePosition.html#Syncfusion_Maui_Core_BadgePosition_BottomLeft),  [`BottomRight`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgePosition.html#Syncfusion_Maui_Core_BadgePosition_BottomRight), `Left`, `Top`, `Right`, and `Bottom` using the [`Position`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.BadgeSettings.html#Syncfusion_Maui_Core_BadgeSettings_Position) property.

File: MAUI/TabView/Getting-Started.md [hunk 3]
```diff
@@ -210,4 +210,250 @@ namespace TabViewMauiSample
...
\ No newline at end of file
+N> View [sample](https://github.com/SyncfusionExamples/maui-tabview-samples/tree/main/TabViewGettingStarted) in GitHub
+
+## Populate ItemsSource
+
+Items can be added to the control using the `ItemsSource` property of [SfTabView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TabView.SfTabView.html).
+
+Using ItemsSource, objects of any class can be given as items for SfTabView. The views corresponding to the objects can be set using the HeaderItemTemplate for the header items and ContentItemTemplate for the content. 
+
+Create a model class with TabItems collection property initialized with required number of data objects   as shown in the following code example.
```
Comment (laksmipriyamurugesan) [inline MAUI/TabView/Getting-Started.md:221]: Create a model class using the `TabItems` collection property that is initialized with the required number of data objects,   as shown in the following code example.

File: MAUI/TabView/Getting-Started.md [hunk 4]
```diff
@@ -210,4 +210,250 @@ namespace TabViewMauiSample
...
 
-N> View [sample](https://github.com/SyncfusionExamples/maui-tabview-samples/tree/main/TabViewGettingStarted) in GitHub
\ No newline at end of file
+N> View [sample](https://github.com/SyncfusionExamples/maui-tabview-samples/tree/main/TabViewGettingStarted) in GitHub
+
+## Populate ItemsSource
+
+Items can be added to the control using the `ItemsSource` property of [SfTabView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TabView.SfTabView.html).
+
+Using ItemsSource, objects of any class can be given as items for SfTabView. The views corresponding to the objects can be set using the HeaderItemTemplate for the header items and ContentItemTemplate for the content.
```
Comment (laksmipriyamurugesan) [inline MAUI/TabView/Getting-Started.md:219]: Objects of any class can be given as items for SfTabView by using ItemsSource,. The views corresponding to the objects can be set using the HeaderItemTemplate for the header items and ContentItemTemplate for the content.

File: MAUI/TabView/Getting-Started.md [hunk 5]
```diff
@@ -210,4 +210,250 @@ namespace TabViewMauiSample
...
+           
+        }
+
+  }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+The following code example binds the collection to the SfTabView.ItemsSource property.
```
Comment (laksmipriyamurugesan) [inline MAUI/TabView/Getting-Started.md:299]: The following code example binds the collection to the `SfTabView.ItemsSource` property.

File: MAUI/TabView/Getting-Started.md [hunk 6]
```diff
@@ -210,4 +210,250 @@ namespace TabViewMauiSample
...
+    }
+  }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Content item template
+
+By defining the `ContentItemTemplate` of the SfTabView, a custom user interface(UI) can be achieved to display the tab content data items.
```
Comment (laksmipriyamurugesan) [inline MAUI/TabView/Getting-Started.md:400]: Please highlight the controls' name.

File: MAUI/Badge-View/Predefined-symbols.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+description: Learn how to set Syncfusion badge view predefined symbols and types of badge icons in .NET MAUI platform
+platform: maui
+control: SfBadgeView
+documentation: ug
+
+---
+
+# Predefined symbols in .NET MAUI Badge View (SfBadgeView)
+
+The badge icon has changed using the `Icon` property. Badge icons will be visible when you do not set the badge text. The badge supports the following types of `Icon`
```
Comment (laksmipriyamurugesan) [inline MAUI/Badge-View/Predefined-symbols.md:14]: The badge icon has been changed using the `Icon` property. Badge icons will be visible when you do not set the badge text. The badge supports the following types of `Icon`.

Keep full stop at the end.

**Pull request merged**